UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web.Tests/UKHO.MaritimeSafetyInformation.Web.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
```

Comment:
Could we make sure that we start with the latest version of the packages please. (.e. Test.Sdk should be 17.1.0, NUnit 3.3.13, NUnitTestAdaptor 4.2.1, etc).

Reply:
Added latest versions packages. Thanks